flink-core/src/main/java/org/apache/flink/util/InstantiationUtil.java

```diff
@@ -466,6 +468,7 @@ public static String checkForInstantiationError(Class<?> clazz) {
         }
     }
 
+    @Nullable
```

Comment:
Minor but would it make sense to introduce the Runtime annotation in a separate ticket as it is used in a bunch of places?

Reply:
Ack, it makes sense to be in this PR then, thank you!